client/app/components/packages/filed-eas/edit.js

```diff
@@ -5,6 +5,18 @@ import SubmittablePackageFormValidations from '../../../validations/submittable-
 import SubmittableCeqrInvoiceQuestionnaireFormValidations from '../../../validations/submittable-ceqr-invoice-questionnaire-form';
 
 export default class PackagesFiledEasEditComponent extends Component {
+  @service
+  store;
+
+  constructor(...args) {
+    super(...args);
+
+    if (this.args.package.ceqrInvoiceQuestionnaires.length < 1) {
+      const newCeqrInvoiceQuestionnaire = this.store.createRecord('ceqr-invoice-questionnaire');
+      this.args.package.ceqrInvoiceQuestionnaires.pushObject(newCeqrInvoiceQuestionnaire);
```

Comment:
This is modeled after what we do for `applicants`, `related-actions`, and `bbls`, where the user adds one record in the frontend and then it's POSTed to the backend when they save, except it's added automatically on page load rather than the user having to add it by pressing a button.

We "could" do this in the backend but we decided to handle it in the frontend to mirror how we POST to other entities like the ones listed above. Always up for other suggestions, though, if we think that there's a better way to handle this in the future!